systemtest/JPSRunTest.py

```diff
@@ -67,8 +69,9 @@ def __init__(self, testnumber, argv0, testdir, utestdir="..", jpsreportdir=""):
     def run_test(self, testfunction, fd=0, *args): #fd==1: make fundamental diagram
         assert hasattr(testfunction, '__call__'), "run_test: testfunction has no __call__ function"
         self.__configure()
-        jpscore = os.path.join(self.trunk, "build", "bin", "jpscore")
-        jpscore_exe = self.__find_executable(jpscore)
+        jpscore_exe = self.__find_executable(self.jpscore)
```

Comment:
Do we really use find_executable? shouldn't we remove it and make an assert that the file given in jpscore exits?

Reply:
No, we don't really need it.
This testing machinery is supposed to tun under Linux. So this "windows hack" is not necessary.

Reply:
Depends, the test should also run on windows as we want to test our binaries on windows too.

Reply:
Let's keep it this way. This Driver needs some refactoring, now that jpsreport needs to be tested as well.
We probably should open another issue/PR for this.

Reply:
From my understanding is `find_executable` not needed, as the executable is given to the script. The function searches  for `jpscore.exe` in the old binary output directory which may be empty.

Reply:
It would be better to retrieve these variables from CMAKE. 
This driver needs refactoring anyway.
Do you want to do it in this PR?